### PR TITLE
Muted conversations are not updating the badge count (WEBAPP-3051)

### PR DESCRIPTION
**Pull request by lipis**

### CODE REVIEW

**Comment by bennycode** [comment]
Do we have a unit test for this or a test case by @mythsunwind or @Yserz ?

File: app/script/view_model/WindowTitleViewModel.coffee [hunk 1]
```diff
@@ -35,7 +35,8 @@ class z.ViewModel.WindowTitleViewModel
 
       @conversation_repository.conversations_unarchived().forEach (conversation_et) ->
         if conversation_et.type() isnt z.conversation.ConversationType.CONNECT
```
Comment (gregor) [inline app/script/view_model/WindowTitleViewModel.coffee]: We can move all of this into one line:

`if conversation_et.number_of_unread_messages() and not conversation_et.is_request() and not conversation_et.is_muted()`

Comment (lipis) [inline app/script/view_model/WindowTitleViewModel.coffee]: that's how it was.. but it was a bit longer that we wanted to 💃

Comment (bennycode) [inline app/script/view_model/WindowTitleViewModel.coffee]: ?

Comment (lipis) [inline app/script/view_model/WindowTitleViewModel.coffee]: we took it offline :)

**Comment by lipis** [comment]
Changed 1 line of code.. and then 200 lines for testing :D Totally worth it.. 💃 👯 🌾 🍹

**Pull request merged**